backend/src/packages/chaiNNer_standard/image_dimension/utility/get_bounding_box.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Tuple
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import NumberOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import utility_group
+
+
+@utility_group.register(
+    schema_id="chainner:image:get_bbox",
+    name="Get Bounding Box",
+    description="Gets a bounding box (X, Y, Height, and Width) of the white area of a mask.",
+    icon="BsRulers",
+    inputs=[
+        ImageInput(channels=1),
+        SliderInput(
+            "Threshold",
+            precision=1,
+            minimum=1,
```

Comment:
Please specify the maximum. I don't have the default values for all inputs in my head all the time.

```suggestion
            minimum=1,
            maximum=100,
```